contracts/base/AuctioneerManaged.sol

```diff
@@ -0,0 +1,30 @@
+pragma solidity ^0.4.11;
+
+contract AuctioneerManaged {
+    // auctioneer has the power to manage some variables
+    address public auctioneer;
+    
+    function AuctioneerManaged () public {
```

Comment:
DX upgrade has no initializer, but AuctioneerManaged has one? and its inherited.

Do you know what is happening there under the hood?